wheelhouse.txt

```diff
@@ -10,8 +10,13 @@ Jinja2==2.10;python_version >= '3.0' and python_version <= '3.4' # py3 trusty
 Jinja2==2.11;python_version == '2.7' or python_version == '3.5'  # py27, py35
 Jinja2;python_version >= '3.6' # py36 and on
 
+# Cython is required to build PyYAML. To find out the supported versions check
+# https://github.com/cython/cython/issues/2800
+Cython
```

Comment:
Not 100% sure because of this change, but a reactive charm has failed to build over the weekend. It's failing at Cython installation due to missing (Python.h: python3-dev).

Are you going to manage this dependency centrally or do you expect every single charm to have python3-dev?

```
2023-04-10 07:13:03.353319 | focal-medium | ::             x86_64-linux-gnu-gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -I/root/snap/charmcraft/common/tmp8fu559sx/include -I/usr/include/python3.10 -c /root/snap/charmcraft/common/pip-install-s2qs0rbg/cython_6d98672645614f43bf70d0b7f222b29d/Cython/Plex/Scanners.c -o build/temp.linux-x86_64-cpython-310/root/snap/charmcraft/common/pip-install-s2qs0rbg/cython_6d98672645614f43bf70d0b7f222b29d/Cython/Plex/Scanners.o
2023-04-10 07:13:03.353337 | focal-medium | ::             /root/snap/charmcraft/common/pip-install-s2qs0rbg/cython_6d98672645614f43bf70d0b7f222b29d/Cython/Plex/Scanners.c:21:10: fatal error: Python.h: No such file or directory
2023-04-10 07:13:03.353350 | focal-medium | ::                21 | #include "Python.h"
2023-04-10 07:13:03.353363 | focal-medium | ::                   |          ^~~~~~~~~~
2023-04-10 07:13:03.353375 | focal-medium | ::             compilation terminated.
2023-04-10 07:13:03.353640 | focal-medium | ::             error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
2023-04-10 07:13:03.353668 | focal-medium | ::             [end of output]
2023-04-10 07:13:03.353683 | focal-medium | ::
2023-04-10 07:13:03.353715 | focal-medium | ::         note: This error originates from a subprocess, and is likely not a problem with pip.
2023-04-10 07:13:03.353737 | focal-medium | ::       error: legacy-install-failure
2023-04-10 07:13:03.353748 | focal-medium | ::
2023-04-10 07:13:03.353759 | focal-medium | ::       × Encountered error while trying to install package.
2023-04-10 07:13:03.353770 | focal-medium | ::       ╰─> Cython
```